source/ui/beamline/AMExtendedControlEditor.cpp

```diff
@@ -456,6 +462,20 @@ QSize AMExtendedControlEditor::sizeHint() const{
 	return newHint;
 }
 
+void AMExtendedControlEditor::onControlMinimumValueChanged()
+{
+	if (control_) {
```

Comment:
When you pass it in after it has connected the correct value is obtained from the control_->minimumValue()

essentially the logic we'd end up with (simplified) in the editors is:
```
setControl(control):
...
    if (control is connected)
        maxValue_ = control->maxValue()

    connect(control, maxValueChanged(double), this, onControlMaxValueChanged(double))
...
onControlMaxValueChanged(newMaxValue):
    maxValue_ = newMaxValue
```